client/blocks/signup-form/index.jsx

```diff
@@ -931,6 +932,14 @@ class SignupForm extends Component {
 	}
 }
 
+function TrackRender( { children, eventName } ) {
+	useEffect( () => {
+		analytics.tracks.recordEvent( eventName );
+	}, [] );
```

Comment:
Do we need the empty dependency array here? In VSCode I'm getting an ESLint warning on this line: `React Hook useEffect has a missing dependency: 'eventName'. Either include it or remove the dependency array.eslint(react-hooks/exhaustive-deps)`

Reply:
Totally right! I just wanted the canonical way to only run the effect once, but `eventName` could technically change. Fewer future foot guns and all that.
Will have to figure out why my VS Code isn't showing squiggles :'(